kolibri/plugins/management/assets/src/views/user-page/index.vue

```diff
@@ -212,6 +214,7 @@
       getters: {
         users: state => state.pageState.facilityUsers,
         modalShown: state => state.pageState.modalShown,
+        currentUserId: state => state.core.session.user_id,
```

Comment:
There's a core getter `currentUserId` that you can use here.

Reply:
Beautiful.